Polyhedron/demo/Polyhedron/Plugins/Mesh_3/C3t3_io_plugin.cpp

```diff
@@ -19,9 +19,11 @@ class Polyhedron_demo_c3t3_binary_io_plugin :
 
 public:
   QString name() const override { return "C3t3_io_plugin"; }
-  QString nameFilters() const override { return "binary files (*.cgal);;ascii (*.mesh);;maya (*.ma)"; }
-  QString saveNameFilters() const override { return "binary files (*.cgal);;ascii (*.mesh);;maya (*.ma);;avizo (*.am);;OFF files (*.off)"; }
-  QString loadNameFilters() const override { return "binary files (*.cgal);;ascii (*.mesh)"; }
+  QString nameFilters() const override { return "binary files (*.cgal);;ascii (*.mesh);;maya (*.ma);;avizo (*.tetra.am)"; }
+  QString saveNameFilters() const override{
+    return "binary files (*.cgal);;ascii (*.mesh);;maya (*.ma);;avizo (*.tetra.am);;OFF files (*.off)"; }
```

Comment:
Can we really save to `.tetra.am` files?

Reply:
yes we can!

Reply:
using `CGAL::IO::output_to_avizo()`